gateway/src/main/java/com/lyft/data/gateway/router/RoutingManager.java

```diff
@@ -32,7 +32,8 @@
  */
 @Slf4j
 public abstract class RoutingManager {
-  private static final AtomicLong requestCounter = new AtomicLong(0);
```

Comment:
Is there existing consumer of requestCounter and is it ok to just remove the counter.